packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiRequest.kt

```diff
@@ -90,16 +92,24 @@ class FlutterApiRequest {
             }
         }
 
+        @JvmStatic
+        fun checkForEmptyBodyIfRequired(options: RestOptions, methodName: String): Void? {
```

Comment:
nit: May omit the return type `Void?` so it returns `Unit` (not anything meaningful) by default in Kotlin, then can omit the `return` statement.

Reply:
good suggestion, done for latest revision